### ISSUE TITLE
FileUpload widget

### ISSUE THREAD

**Issue by lassoan** [jupyter-xeus/xeus-python#268]
We are switching over to using xeus-python in [3D Slicer](https://www.slicer.org/) and we are very happy with it so far. Thank you very much for developing this library!

I have tried to add a FileUpload iPython widget to a notebook so that we can set up processing workflow that users can apply on their own data. However, FileUpload widget does not return any data. I'm not sure if this is expected or I'm doing something wrong.

How to reproduce:

Run this in a cell:

```python
from ipywidgets import FileUpload
upload = FileUpload()
upload
```

then select a file to upload. After this, `upload.data` and `upload.metadata` fields are set correctly when a regular Python3 kernel is used, but they are empty when xeus-python kernel is used.

**Comment by SylvainCorlay** [comment]
Thanks for reporting. I will look into it this week.

**Comment by lassoan** [comment]
Thanks a lot for the quick fix. Upload of small files now works well!

Unfortunately, larger files (few ten megabytes) still fails the same way as before. For example, using [these data sets](https://1drv.ms/u/s!Arm_AFxB9yqHu5J5GV9xK09QGX5ODA?e=nzhwq9): Small.vtk (630KB) works, Large.vtk (17MB) does not work.

**Comment by SylvainCorlay** [comment]
Note: the upload widget was greatly improved for ipywidgets. 8.0 (to be released).

I don't know what are the limitations with respect to large files though.

**Comment by SylvainCorlay** [comment]
We should check with the classic Python 3 kernel to check if this is a xeus-python issue.

**Comment by lassoan** [comment]
Very good point! I have tried with a standard Python 3 kernel and it has the same issue - works with the small test file and does not work with the "large" (still fairly small - 17MB) file.

I did some more search and it turned out that it is a limitation imposed on maximum websocket message size and the limit can be increased as described here: https://github.com/jupyter/notebook/issues/3468

I've increased the limit and it indeed fixed the problem. I've managed to upload large files using FileUpload widget using xeus-python kernel.

It could be nice if the kernel reported an error message
